climakitae/core/data_interface.py

```diff
@@ -1749,25 +1924,24 @@ def get_data_options(
 
     Parameters
     ----------
-    variable: str, optional
+    variable: str (optional)
```

Comment:
`, optional` is styled better. Something worth cleaning up probably only when convenient.